bundles/rocky9.1/packages.txt.gotmpl

```diff
@@ -15,17 +14,21 @@ yum-utils
 cloud-init
 cloud-utils-growpart
 container-selinux
+{{ if .FetchKubernetesRPMs -}}
 kubectl-{{ .KubernetesVersion }}-0
 kubelet-{{ .KubernetesVersion }}-0
 kubeadm-{{ .KubernetesVersion }}-0
+cri-tools
+{{- end }}
 conntrack
 ebtables
 ethtool
 iproute
 iptables
 socat
-cri-tools
 gcc
 libseccomp
 nfs-utils
 sssd-kcm
+iptables-libs
```

Comment:
why these additional rpms added?

Reply:
this was failing on ebtables install with the current AMI